Atomic/atomic.py

```diff
@@ -79,13 +79,25 @@ def docker_binary(self):
             self.docker_cmd = util.default_docker()
         return self.docker_cmd
 
-    def get_label(self, label, image=None):
+    def get_labels(self, image=None):
```

Comment:
Here again, would prefer the use of the labels from the image or container object.